components/member-profile/index.js

```diff
@@ -8,12 +10,20 @@ const Profile = (props) => {
   const { data } = useFetch(getDataURL(props.id));
   return (
     <>
-      <img src={imgLink} className={classNames.profilePic} />
+      <img src={imgLink} className={classNames.profilePic} alt="profile" />
```

Comment:
alt="profile image"

Reply:
good suggestion. 
Due to the linting errors set on the project, its giving error whein i give alt name as "profile image"
`error  Redundant alt attribute. Screen-readers already announce `img` tags as an image. You don’t need to use the words `image`, `photo,` or `picture` (or any specified custom words) in the alt prop`

Reply:
Thats the reason the alt name is profile and not profile image